osbs/utils.py

```diff
@@ -327,20 +328,34 @@ def make_name_from_git(repo, branch, limit=53, separator='-', hash_size=5):
 
     size = (2 * len(separator)) + hash_size
 
+    disallowed = '._'
```

Comment:
Agree. we now that `'-' + string.ascii_lowercase + string.digits` is allowed. Anything that is not in that should be removed or replaced with something.